alpaca/rest.go

```diff
@@ -341,7 +342,7 @@ func (c *Client) ListBarLists(symbols []string, opts BarListParams) ([]BarList,
 		vals.Set("limit", strconv.FormatInt(int64(*opts.Limit), 10))
 	}
 
-	u, err := url.Parse(fmt.Sprintf("%v/v1/bars?%v", base, vals.Encode()))
+	u, err := url.Parse(fmt.Sprintf("%sv1/bars/%s?%v", dataUrl, opts.Timeframe, vals.Encode()))
```

Comment:
How about `path.Join()`? dataUrl is fine but people could forget slash often in env var.

Reply:
None of the other methods use join - will add the / to match the others. If we want to change all of them to use join we can do that in a new PR.